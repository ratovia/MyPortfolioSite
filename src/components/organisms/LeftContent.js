import React from "react"
import styled from "styled-components"
import { Profile } from "../molecules/Profile"

const ContentArea = styled.div`
  width: 100%;
  height: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
`
export const LeftContent = ({ profileText, skillText, imageUrl }) => {
  return (
    <ContentArea>
      <Profile
        profileText={profileText}
        skillText={skillText}
        imageUrl={imageUrl}
      ></Profile>
    </ContentArea>
  )
}
