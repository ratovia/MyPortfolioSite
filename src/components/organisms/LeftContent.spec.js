import React from "react"
import renderer from "react-test-renderer"
import { LeftContent } from "./LeftContent"
import javascriptIcon from "../../stories/assets/javascript_icon.png"

describe("LeftContent", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <LeftContent
          profileText="Hello world"
          skillText="This is Component"
          imageUrl={javascriptIcon}
        />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
