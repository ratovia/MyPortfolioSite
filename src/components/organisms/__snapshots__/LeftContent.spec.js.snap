// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftContent renders correctly 1`] = `
<div
  className="sc-furvIG gMDNLi"
>
  <div
    className="sc-jRQAMF jFTfeO"
  >
    <div
      className="sc-iCfLBT eyXoBR"
    >
      <img
        alt="アイコン"
        className="sc-bdvvaa hVPPTo"
        height="240px"
        src="test-file-stub"
        width="240px"
      />
    </div>
    <div
      className="sc-gKckTs epwxeJ"
    >
      <h6
        className="sc-eCImvq kPbozt"
      >
        <p
          className="sc-gsDJrp jWZtA"
          color="white"
        >
          Hello world
        </p>
      </h6>
    </div>
    <div
      className="sc-gKckTs epwxeJ"
    >
      <h6
        className="sc-eCImvq kPbozt"
      >
        <p
          className="sc-gsDJrp jWZtA"
          color="white"
        >
          This is Component
        </p>
      </h6>
    </div>
  </div>
</div>
`;
